ktor-server/ktor-server-test-host/api/ktor-server-test-host.api

```diff
@@ -2,6 +2,7 @@ public final class io/ktor/server/testing/ApplicationTestBuilder : io/ktor/serve
 	public fun <init> ()V
 	public fun createClient (Lkotlin/jvm/functions/Function1;)Lio/ktor/client/HttpClient;
 	public fun getClient ()Lio/ktor/client/HttpClient;
+	public final fun startApplication ()V
```

Comment:
Could you tell me why should we add new API in the patch release?

Reply:
thanks, I changed the base branch